transports/quic/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## 0.9.2 - unreleased
 
 - Cut stable release.
+- Switch from webpki to rustls-webpki.
+  See [PR 4381].
+
+[PR 4381]: https://github.com/libp2p/rust-libp2p/pull/4381
```

Comment:
This entry should go into `transports/tls/CHANGELOG.md`.

Reply:
Oh, absolutely, my bad